### PR TITLE
LPS-116002 Change Audit Graph empty message to include documentation link

### PR DESCRIPTION
**Pull request by dgarciasarai**
**Feature flag: YES**

The Audit Graph is hidden. The configuration file to show the chart is here: https://issues.liferay.com/browse/LPS-115805

**Module**

```
~/liferay-portal/modules/apps/content-dashboard/content-dashboard-web
```

* Global Menu > Content Dashboard

**Screenshots**

Screenshot 2020-07-09 at 09 38 00

cc: @bryceosterhaus @wincent

### CODE REVIEW

**Comment by dgarciasarai** [comment]
ci:test:sf

**Comment by dgarciasarai** [comment]
ci:test:relevant

File: modules/apps/content-dashboard/content-dashboard-web/src/main/resources/META-INF/resources/js/components/EmptyAuditBarChart.js [hunk 1]
```diff
@@ -26,24 +24,29 @@ import {
...
+					<div
+						dangerouslySetInnerHTML={{
+							__html: Liferay.Util.sub(
+								Liferay.Language.get(
+									'learn-how-to-tailor-categories-to-your-needs'
+								),
+								`<a href=${LEARN_HOW_URL} target="_blank">`,
+								'</a>'
+							),
+						}}
```
Comment (wincent) [inline modules/apps/content-dashboard/content-dashboard-web/src/main/resources/META-INF/resources/js/components/EmptyAuditBarChart.js:45]: For reference, [Slack thread on this](https://liferay.slack.com/archives/CNBG06JS3/p1594204672332000).

Was suggested that you could do this without `dangerouslySetInnerHTML` ([codesandbox](https://codesandbox.io/s/broken-violet-w3yb2?file=/src/App.js)), so if you're sure that's not going to work, I'm fine with this; there is [at least one other example that does it the same way you're doing it](https://github.com/liferay/liferay-portal/blob/dfa080768228be927b37085a56e541c05ce0f072/modules/apps/redirect/redirect-web/src/main/resources/META-INF/resources/js/DestinationUrlInput.js#L112-L118) already in liferay-portal.

**Comment by wincent** [comment]
ci:forward

File: modules/apps/content-dashboard/content-dashboard-web/src/main/resources/META-INF/resources/js/components/EmptyAuditBarChart.js (hunk 1 above)
Comment (dgarciasarai) [reply modules/apps/content-dashboard/content-dashboard-web/src/main/resources/META-INF/resources/js/components/EmptyAuditBarChart.js:45]: I tried @bryceosterhaus suggestion, but the portal was rendering the HTML as a plain text 😢

**Pull request closed without merging**